include/private/entry.h

```diff
@@ -165,21 +207,158 @@ new_entry( enum stumpless_facility facility,
            char *message,
            size_t message_length );
 
+/**
+ * Appends the application name from a Stumpless entry to a string builder.
+ *
+ * This function takes the application name from the provided Stumpless entry and
+ * appends it to the specified string builder. It is designed for building strings
+ * that include the application name of log entries.
+ *
+ * **Thread Safety: MT-Unsafe**
+ * This function is not thread-safe as it accesses shared data (the entry's application name)
+ * without synchronization mechanisms.
+ *
+ * **Async Signal Safety: AS-Unsafe**
+ * Unsafe to call from asynchronous signal handlers due to memory manipulation
+ * and accessing shared data structures.
+ *
+ * **Async Cancel Safety: AC-Unsafe**
+ * Not safe in contexts of asynchronous cancellation, as it might lead to inconsistent
+ * states of shared data.
+ *
+ * @since release 1.0.0
```

Comment:
Remove this `@since` tag unless you've done the research to track down when this function appeared.